### ISSUE TITLE
Ionic cordova Run Android - Failed to apply plugin [class 'com.google.gms.googleservices.GoogleServicesPlugin']

### ISSUE THREAD

**Issue by shaunsideclick** [phonegap/phonegap-plugin-push#2593]
### Expected Behaviour
when running command:  "ionic cordova run android", the build should succeed and run on an AVD or physical mobile device.

### Actual Behaviour
The build fails when running the command:  "ionic cordova run android"
Error:
* Where:
Script 'C:\Mobile APPS\Iyeza Client APP\platforms\android\phonegap-plugin-push\patient-push.gradle' line: 35

* What went wrong:
A problem occurred evaluating project ':app'.
> Failed to apply plugin [class 'com.google.gms.googleservices.GoogleServicesPlugin']

#### Steps to Reproduce 

1. Install phonegap-plugin-push:
    $ ionic cordova plugin add phonegap-plugin-push
    $ npm install --save @ionic-native/push

2. run command: ionic cordova platform remove android

3. run command: ionic cordova platform add android

4. run command: ionic cordova run android

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
Cordova Platforms     : android 7.0.0
Ionic Framework    : ionic-angular 3.9.2

#### (Android) What device vendor (e.g. Samsung, HTC, Sony...)
- Nokia 3 (Physical mobile device)
- AVD (Android 8.0 (Google APIs))

#### Cordova CLI version and cordova platform version

    cordova --version                                    8.0.0
    cordova platform version android           7.0.0

#### Plugin version

    cordova plugin version | grep phonegap-plugin-push   ^4.16.0

#### Sample Push Data Payload
N/A

#### Sample Code that illustrates the problem

N/A

#### Logs taken while reproducing problem

Please see the CMD output in the following link:
https://gist.github.com/shaunsideclick/c08d9812360bd5651629cb7ae30d5672

**Comment by jcesarmobile** [comment]
the plugin version you provided is not valid, looks like the ionic-native/push version and not the phonegap-plugin-push version.

Anyway, if you are using cordova-android 7.0.0, you won't get latest version of the plugin as it requires cordova-android 7.1.x, so update your platform to latest

**Comment by shaunsideclick** [comment]
Hi

Thank you for your response. I upgraded cordova-android 7.0.0 about 15 min
ago and the build still failed.
I then commented out the "apply plugin:
com.google.gms.googleservices.GoogleServicesPlugin" in the
cordova-support-google-services build.gradle file and could then do ionic
cordova run android.

Busy testing it now and implementing.

I will keep you updated.

Thank you so much for your feedback.
Shaun
